src/stars.css

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__item {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 > .stars__item:nth-child(-n + 4) {
+  background-image: url("images/star-active.svg");
+}
```

Comment:
all stars classes should work like on stars hw
.stars--1 - just one active star
.stars--4 -4 active stars